e2e/testcafe-devextreme/tests/accessibility/common/list/itemDeleting.ts

```diff
@@ -0,0 +1,33 @@
+import { Properties } from 'devextreme/ui/list.d';
+import { Options } from '../../../../helpers/generateOptionMatrix';
+import url from '../../../../helpers/getPageUrl';
+import { Configuration, testAccessibility } from '../../../../helpers/accessibility/test';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../../container.html'));
+
+const simpleItems = ['Item_1', 'Item_2', 'Item_3'];
+
+const optionsWithSimpleItems: Options<Properties> = {
+  dataSource: [simpleItems],
+  height: [400],
+  grouped: [false],
+  searchEnabled: [true],
+  allowItemDeleting: [true, false],
+  itemDeleteMode: ['toggle', 'context', 'slideButton', 'slideItem', 'static', 'swipe'],
```

Comment:
Do we really need to test all this modes? Is there any change in markup? I guess one prop could be enough

Reply:
Now testing time for all configuration is a bit more 1 minute and I think it's not critical have addional 12 tests for deletingMode markup variants